### ISSUE TITLE
When trying to install in Python 3.10.5 it fails

### ISSUE THREAD

**Issue by fbrodrigorezino** [urllib3/urllib3#2724]
### Subject

Getting error while installing in Python 3.10.5

### Environment

Describe your environment.
At least, paste here the output of:

Python 3.10.5
Monterey 12.5

### Steps to Reproduce

- Use python 3.10.5
- run `pip install 1.26.12`

### Expected Behavior

Install the package

### Actual Behavior

```
Command ['/Users/****/Library/Caches/pypoetry/virtualenvs/fresh-tests-toolbelt-XPs3JBYc-py3.10/bin/pip', 'install', '--no-deps', 'urllib3==1.26.12'] errored with the following return code 1, and output: 
  Looking in indexes: https://rodrigo:****@****.jfrog.io/artifactory/api/pypi/pypi/simple
  ERROR: Could not find a version that satisfies the requirement urllib3==1.26.12 (from versions: 0.3, 1.0, 1.0.1, 1.0.2, 1.1, 1.2, 1.2.1, 1.2.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.7.1, 1.8, 1.8.2, 1.8.3, 1.9, 1.9.1, 1.10, 1.10.1, 1.10.2, 1.10.3, 1.10.4, 1.11, 1.12, 1.13, 1.13.1, 1.14, 1.15, 1.15.1, 1.16, 1.17, 1.18, 1.18.1, 1.19, 1.19.1, 1.20, 1.21, 1.21.1, 1.22, 1.23, 1.24, 1.24.1, 1.24.2, 1.24.3, 1.25, 1.25.1, 1.25.2, 1.25.3, 1.25.4, 1.25.5, 1.25.6, 1.25.7, 1.25.8, 1.25.9, 1.25.10, 1.25.11, 1.26.0, 1.26.1, 1.26.2, 1.26.3, 1.26.4, 1.26.5, 1.26.6, 1.26.7, 1.26.8, 1.26.9, 1.26.10, 1.26.11)
  ERROR: No matching distribution found for urllib3==1.26.12
  
  [notice] A new release of pip available: 22.1.2 -> 22.2.2
  [notice] To update, run: pip install --upgrade pip
```

**Comment by sethmlarson** [comment]
I've seen this issue, trying to reproduce locally.

**Comment by sethmlarson** [comment]
I wasn't able to reproduce this issue on Ubuntu with Python 3.10.5 and pip 22.1.2? Could you give try again or give me more information?

```
source venv/bin/activate
python --version
Python 3.10.5

python -m pip install pip==22.1.2
Collecting pip==22.1.2
  Using cached pip-22.1.2-py3-none-any.whl (2.1 MB)
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 22.0.4
    Uninstalling pip-22.0.4:
      Successfully uninstalled pip-22.0.4
Successfully installed pip-22.1.2

python -m pip install --no-deps urllib3==1.26.12
Collecting urllib3==1.26.12
  Using cached urllib3-1.26.12-py2.py3-none-any.whl (140 kB)
Installing collected packages: urllib3
Successfully installed urllib3-1.26.12

[notice] A new release of pip available: 22.1.2 -> 22.2.2
[notice] To update, run: pip install --upgrade pip
(venv) sethmlarson:/tmp$ python -m pip install urllib3==1.26.12
Collecting urllib3==1.26.12
  Downloading urllib3-1.26.12-py2.py3-none-any.whl (140 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 140.4/140.4 KB 2.0 MB/s eta 0:00:00
Installing collected packages: urllib3
Successfully installed urllib3-1.26.12
WARNING: You are using pip version 22.1.2; however, version 22.2.2 is available.
```

**Comment by sethmlarson** [comment]
Oh, you're using a custom index. From the output of your command:

```
Looking in indexes: https://rodrigo:****@****.jfrog.io/artifactory/api/pypi/pypi/simple
```

You'll have to wait for your index to add the new package before using it. Going to close this issue as solved, thanks for reporting!
